_posts/2019-08-28-keyboard-practicing-website-for-everyone.md

```diff
@@ -1,10 +1,14 @@
 ---
 layout: post
-title: "A keyboard practicing website for everyone"
-date: 2019-08-28 13:30:22 +0200
-categories: update
+title:  A keyboard practicing website for everyone
+date:   2019-08-28 13:30:22 +0200
+image:  keyboard.jpg
+author: Emilia Michanek
+tags:   Screen readers
```

Comment:
As mentioned in another comment, we can probably remove tags. :) I'm also not sure how well Jekyll handles prepended spaces, probably just fine, but I thought I'd mention it! ~^o^~

Reply:
Fixed